### DISCUSSION TITLE
Storage Bucket Contents Scrolling Bug (Dashboard)

### DISCUSSION THREAD

**Discussion by hansfuchs** [supabase/supabase#16951] [Contribute to Supabase]
# Bug report

## Describe the bug

I have a bucket called "images" in which common media files are stored. Inside this bucket is a folder (see image, in this case it's called "hl-posts"), in which I want to store related, but different kinds of files. I used the supabase storage API to upload ~1600 images into said subfolder, but when I navigate only the first 199 are shown there. When I scroll all the way down, no additional files are loaded. However, when I continue scrolling in the first column (the bucket "images"), the files that should appear at the bottom of "hl-posts" are loaded instead of the files inside the bucket.

By trying to access the files over the CDN I have confirmed that the files are located inside the folder (and can be accessed without problems), they are simply not loaded in the right column.

Clicking on a media file that is wrongfully displayed in the left column shows no preview.

## Screenshots

The following screenshot should clear up things:
![image](https://user-images.githubusercontent.com/32516822/229577342-30fb12b2-2808-44f4-a8b6-81c0d8a6ccbe.png)

## To Reproduce

1. Create a bucket
2. Add a folder in said bucket
3. Upload >200 files into bucket
4. Upload >200 files into folder inside bucket
5. Try to scroll down inside folder to see it doesn't work
6. Click inside left column (bucket), then scroll down: it should show the contents of the folder INSTEAD of the actual contents of the bucket

## Expected behavior

Scrolling down inside a folder inside a bucket should load the correct files.

## System information

- OS: Windows
- Browser: latest version of Vivaldi (Chromium)
- Version of supabase-js: 2.10.0
- Version of Node.js: 19.0.0

**Comment by lukuser** [comment]
I encounter the same issue. Is there anything in the works for it?

**Comment by pablojsx** [comment]
same issue, only shows 199 files

**Comment by bytemtek** [comment]
Same here+

**Comment by AntonOfTheWoods** [comment]
You can search for items and they will get shown but you can't show more. Pretty amateurish...
